dht-node/src/dht_node/index.ts

```diff
@@ -4,10 +4,21 @@ import DHT from '@hyperswarm/dht'
 import { logger } from '@/server/logger'
 import CONFIG from '@/config'
 import { FederatedCommunity as DbFederatedCommunity } from '@entity/FederatedCommunity'
+import { Community as DbCommunity } from '@entity/Community'
+import DEVOP from '@/config/devop'
```

Comment:
We made on the backend some linting rules to have something like this:
```
import { DEVOP } from '@/config/devop'
```

Reply:
does not work with this notation like the same with the import CONFIG